extensions/mentions/migrations/2022_10_21_000001_alter_created_at_set_current_timestamp_on_post_mentions_group.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+use Illuminate\Database\Schema\Builder;
+
+return [
```

Comment:
let's delete this and have it all in one migration (previous comment code suggestion)